tee-worker/core-primitives/node-api/factory/src/lib.rs

```diff
@@ -20,7 +20,7 @@ use itp_api_client_types::{ParentchainApi, WsRpcClient};
 use sp_core::sr25519;
 
 /// Trait to create a node API, based on a node URL and signer.
-pub trait CreateNodeApi: Send + Sync + 'static {
+pub trait CreateNodeApi {
```

Comment:
This is the reason : it should have been bound by  `Send + Sync + 'static` already -> similar to `EnclaveBase`